src/reduce/problem_description.cpp

```diff
@@ -70,6 +70,22 @@ NetworkConfig ProblemDescriptionExtreme::MakeNetworkConfig() const
     return NetworkConfig{ss.str()};
 }
 
+NetworkConfig ProblemDescriptionExtremeAminmaxBackward::MakeNetworkConfig() const
+{
+    auto input_numel  = xDesc.GetElementSize();
+    auto output_numel = yDesc.GetElementSize();
+    auto dtype        = xDesc.GetType();
+
+    std::ostringstream ss;
+
+    ss << "dtype" << dtype;
+    ss << "input_numel" << input_numel;
+    ss << "output_numel" << output_numel;
```

Comment:
Is `output_numel` necessary ?

Reply:
different dim value leads to different output_numel so its necessary